src/Gui/DlgSettings3DView.ui

```diff
@@ -40,6 +40,25 @@ lower right corner within opened files</string>
         </property>
        </widget>
       </item>
+      <item>
+       <widget class="Gui::PrefCheckBox" name="CheckBox_WbByTab">
+        <property name="toolTip">
+         <string>If checked, FreeCAD will remember which workbench is active for each tab of the viewport</string>
```

Comment:
Werner usually mentions that we should avoid using the name `FreeCAD` in any fixed text. The reason is that this breaks the branding of the program, in case anybody would like to create their own version of the program (a commercial fork for example). So, we need to use a neutral word, like "application" or "program".

`If checked, the program will remember which workbench was active for each tab of the viewport`